testbed/tests/test_styling.py

```diff
@@ -0,0 +1,6 @@
+from types import NoneType
+
+
+async def test_dark_mode_state_read(app):
```

Comment:
This should be part of `test_app.py`, not a standalone test file for a single property.